robot/Cumulus/tests/browser/contacts_accounts/deletebehaviors_household.robot

```diff
@@ -1,19 +1,30 @@
 *** Settings ***
 
 Resource        robot/Cumulus/resources/NPSP.robot
-Suite Setup     Open Test Browser
+Library         cumulusci.robotframework.PageObjects
+...             robot/Cumulus/resources/ContactPageObject.py
+...             robot/Cumulus/resources/NPSP.py
+Suite Setup     Run keywords
+...             Open Test Browser
+...             Setup Test Data
 Suite Teardown  Delete Records and Close Browser
 
+***Keywords***
+Setup Test Data
+    &{contact} =              API Create Contact        Email=automation@robot.com 
```

Comment:
It's better to use `@example.com` - that domain is specifically set up for documentation and code examples. robot.com is an actual domain that belongs to somebody.